src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -66,11 +82,11 @@ public function supportsNormalization($data, $format = null, $context = []): boo
             return false;
         }
 
-        return $data instanceof ProductVariantInterface && $this->isNotAdminGetOperation($context);
+        return $data instanceof ProductVariantInterface && $this->isNotAdminGetOperation();
     }
 
-    private function isNotAdminGetOperation(array $context): bool
+    private function isNotAdminGetOperation(): bool
```

Comment:
Now, the method name could be changed to sth like:
```suggestion
    private function isNotAdminApiSection(): bool
```